keyboards/niu_mini/keymaps/yttyx/config.h

```diff
@@ -0,0 +1,12 @@
+#ifndef CONFIG_H
```

Comment:
You can replace lines 1 and 2 with just `#pragma once`

Reply:
Making this change created an issue whereby no layer transitions occurred. Please see general comment for more information.

Reply:
That may be because you have `NO_ACTION_TAPPING` defined.

Reply:
Hi drashna. Many thanks!  I made the check, and as you suggest, `CONFIG_H` _is_ already defined. I have amended `config.h` accordingly and successfully retested. The change is pushed.